assets/scripts/app/__tests__/NotificationBar.test.js

```diff
@@ -1,81 +1,72 @@
+      expect(wrapper.container.firstChild).toBeNull()
     })
 
     it('renders nothing if notification is the empty object', () => {
-      const wrapper = mountWithIntl(<NotificationBar locale="en" notification={{}} />)
-      expect(wrapper.html()).toEqual(null)
+      const wrapper = renderWithIntl(<NotificationBar locale="en" notification={{}} />)
+      expect(wrapper.container.firstChild).toBeNull()
     })
 
     it('renders nothing if notification’s display property is false', () => {
       const notification = { ...TEST_NOTIFICATION, display: false }
-      const wrapper = mountWithIntl(<NotificationBar locale="en" notification={notification} />)
-      expect(wrapper.html()).toEqual(null)
+      const wrapper = renderWithIntl(<NotificationBar locale="en" notification={notification} />)
+      expect(wrapper.container.firstChild).toBeNull()
     })
 
     it('renders nothing if notification’s display property is true but has no other properties', () => {
       const notification = { display: true }
-      const wrapper = mountWithIntl(<NotificationBar locale="en" notification={notification} />)
-      expect(wrapper.html()).toEqual(null)
+      const wrapper = renderWithIntl(<NotificationBar locale="en" notification={notification} />)
+      expect(wrapper.container.firstChild).toBeNull()
     })
 
     it('renders nothing if the locale is not English', () => {
       const notification = { display: true }
-      const wrapper = mountWithIntl(<NotificationBar locale="de" notification={notification} />)
-      expect(wrapper.html()).toEqual(null)
+      const wrapper = renderWithIntl(<NotificationBar locale="de" notification={notification} />)
+      expect(wrapper.container.firstChild).toBeNull()
     })
   })
 
   describe('conditions that render something', () => {
-    it('renders an entire notification', () => {
-      const wrapper = mountWithIntl(<NotificationBar locale="en" notification={TEST_NOTIFICATION} />)
-      expect(wrapper.find('.notification-bar-intro').text()).toEqual(TEST_NOTIFICATION.lede)
-      expect(wrapper.find('.notification-bar-text').text()).toEqual(TEST_NOTIFICATION.text)
-      expect(wrapper.find('a').prop('href')).toEqual(TEST_NOTIFICATION.link)
-      expect(wrapper.find('a').text()).toEqual(TEST_NOTIFICATION.linkText)
-    })
-
     it('renders only the lede', () => {
       const notification = {
         display: true,
         lede: TEST_NOTIFICATION.lede
       }
-      const wrapper = mountWithIntl(<NotificationBar locale="en" notification={notification} />)
-      expect(wrapper.find('.notification-bar-intro').text()).toEqual(notification.lede)
-      expect(wrapper.find('.notification-bar-text').exists()).toEqual(false)
-      expect(wrapper.find('a').exists()).toEqual(false)
+      const wrapper = renderWithIntl(<NotificationBar locale="en" notification={notification} />)
+      expect(wrapper.asFragment()).toMatchSnapshot()
```

Comment:
I think I would opt to testing just the text and not matching the snapshot. 
With the snapshot you lock the notification down to the specific implementation. 
But if you test the text of the notifications it tests much less testing implementation details. 

I would test if:
`'Heads up!'` and
`'Streetmix will be offline for maintainance on January 1, 2025 at 19:00 GMT.',`
are on the page.